theia-php-docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG NODE_VERSION=12.18.3
+ARG NODE_VERSION=12.14.1
```

Comment:
@oscarnevarezleal why is the downgrade necessary?

Reply:
My bad, I was switching between multiple versions. No downgrade is necessary.